http/src/handler.rs

```diff
@@ -562,10 +562,45 @@ impl<M: Metadata, S: Middleware<M>> RpcHandler<M, S> {
 	/// Returns true if the `content_type` header indicates a valid JSON
 	/// message.
 	fn is_json(content_type: Option<&header::HeaderValue>) -> bool {
-		match content_type.and_then(|val| val.to_str().ok()) {
-			Some("application/json") => true,
-			Some("application/json; charset=utf-8") => true,
+		match content_type.and_then(|val| val.to_str().map(str::to_lowercase).ok()) {
+			Some(ref content)
+				if content == "application/json"
+					|| content == "application/json; charset=utf-8" =>
```

Comment:
Is the space relevant here? Should we match against no-space variant as well?

Reply:
the spec doesn't mention spaces, but to be on the safe side, let's match without spaces